Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -225,4 +237,28 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         case .line(let left): bodyField.replace(left: left, right: nil, atLineStart: true)
         }
     }
+    
+    // MARK: UIImagePickerControllerDelegate
+    
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String : Any]) {
+        guard let image = info[UIImagePickerControllerOriginalImage] as? UIImage else { return }
```

Comment:
Would this allow video too? If not, let’s show it to the user? Or maybe show the picker w/o video